src/Blocks/custom/featured-posts/featured-posts.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * Template for the Featured Posts view.
+ *
+ * @package Unicorns
+ */
+
+use UnicornsVendor\EightshiftLibs\Helpers\Components;
+
+$manifest = Components::getManifest(__DIR__);
+
+$blockClass = Components::checkAttr('blockClass', $attributes, $manifest);
+$initialLoad = Components::checkAttr('loadOnStart', $attributes, $manifest) ? 'true' : 'false';
+
+?>
+
+<?php
+echo wp_kses_post(
```

Comment:
as I see you are loading initial content using JS correct?

If yes then this is not the best approach because this way google will not find anything on the page regrading this content so it is bad for the Seo.

You initial content should be loaded using server side rendering (php) and load more should as it is named. Load more

Reply:
Yup, I'm loading it with JS.

What's the best way to render it server-side without throttling the site with a REST request?

I can't really use `WP_Query` like in the stock Featured posts block because I'm fetching data from the local REST api, not from the DB.